docs/getting_started/dbt-airflow-concepts.rst

```diff
@@ -0,0 +1,34 @@
+|                |              |                                                                                 | dbt can run a subset of tasks assuming upstream tasks were run.             | https://docs.getdbt.com/docs/introduction                                            |
++----------------+--------------+---------------------------------------------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------------------------------------------------------+
+| Task           | Node         | Step within a pipeline (DAG or workflow)                                        | In dbt, users write mostly SQL and YML to define the steps of a pipeline.   | https://docs.getdbt.com/reference/node-selection/syntax                              |
+|                |              |                                                                                 | Airflow expects steps to be written in Python.                              | https://airflow.apache.org/docs/apache-airflow/2.7.1/core-concepts/tasks.html        |
++----------------+--------------+---------------------------------------------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------------------------------------------------------+
+| Variables      | Variables    | Key-value configuration that can be used in steps and avoids hard-coded values  |                                                                             | https://docs.getdbt.com/docs/build/project-variables                                 |
+|                |              |                                                                                 |                                                                             | https://airflow.apache.org/docs/apache-airflow/2.7.1/core-concepts/variables.html    |
++----------------+--------------+---------------------------------------------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------------------------------------------------------+
+| Templating     | Macros       | Jinja templating used to access variables, configuration and reference steps    | dbt encourages using jinja templating for control structures (if and for).  | https://docs.getdbt.com/docs/build/jinja-macros                                      |
+|                |              |                                                                                 | Airflow usage is limited to variables, macros and filters                   | https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html             |
```

Comment:
I would say here that because Airflow is Python based, templating is supported natively